Sources/App/Core/PackageContributors.swift

```diff
@@ -0,0 +1,132 @@
+// Copyright 2020-2022 Dave Verwer, Sven A. Schmidt, and other contributors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import Foundation
+import ShellOut
+import Vapor
+
+
+struct PackageAuthors : Encodable, Decodable, Equatable {
+    var authors : [Author]
+    var numberOfContributors : Int
+}
```

Comment:
nit: space before colon shouldn't exist

Very, **very**, picky but it's valuable to have consistency in the way we write our code to make it as easy as possible for others to read and pick up on common patterns.

Additionally, you can use `Codable` which is simply a typealias for a type which is both `Encodable` and `Decodable`.

```suggestion
struct PackageAuthors: Codable, Equatable {
    var authors: [Author]
    var numberOfContributors: Int
}
```